react/tic-tac-toe/src/app/index.js

```diff
@@ -1,11 +1,30 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { func } from 'prop-types';
+
+import actionsCreators from '../redux/auth/actions';
 
 import Game from './screens/Game';
 
-import '../scss/application.scss';
+class App extends Component {
+  componentDidMount() {
+    this.props.authInit();
```

Comment:
This whole component should hsave the src/index render logic. This shouldnt be only for the Game screen, but for the app starting point